test/e2e/flowaggregator_test.go

```diff
@@ -532,6 +504,107 @@ func testHelper(t *testing.T, data *TestData, podAIPs, podBIPs, podCIPs, podDIPs
 			checkRecordsForFlows(t, data, podAIPs.ipv4.String(), svcC.Spec.ClusterIP, isServiceIPv6, false, true, false, false, checkBandwidth)
 		}
 	})
+
+	// Antctl tests ensure antctl is available in a Flow Aggregator Pod
+	// and check the output of antctl commands.
+	t.Run("Antctl", func(t *testing.T) {
+		skipIfNotRequired(t, "mode-irrelevant")
+		flowAggPod, err := data.getFlowAggregator()
+		if err != nil {
+			t.Fatalf("Error when getting flow-aggregator Pod name: %v", err)
+		}
+		podName := flowAggPod.Name
+		for _, args := range antctl.CommandList.GetDebugCommands(runtime.ModeFlowAggregator) {
+			command := []string{}
+			if testOptions.enableCoverage {
+				antctlCovArgs := antctlCoverageArgs("antctl-coverage")
+				command = append(antctlCovArgs, args...)
+			} else {
+				command = append([]string{"antctl", "-v"}, args...)
+			}
+			t.Logf("Run command: %s", command)
+
+			t.Run(strings.Join(command, " "), func(t *testing.T) {
+				stdout, stderr, err := runAntctl(podName, command, data)
+				if err != nil {
+					t.Errorf("Error when running 'antctl %s' from %s: %v\n%s", args, podName, err, antctlOutput(stdout, stderr))
+				}
+			})
+		}
+		t.Run("GetFlowRecordsJson", func(t *testing.T) {
+			checkAntctlGetFlowRecordsJson(t, data, podName, podAIPs, podBIPs, isIPv6)
+		})
+	})
+}
+
+func checkAntctlGetFlowRecordsJson(t *testing.T, data *TestData, podName string, podAIPs, podBIPs *PodIPs, isIPv6 bool) {
+	// A shorter iperfTime that provides stable test results, at which the first record ready in the AggregationProcess but not sent.
+	const iperfTimeSecShort = 5
+	var cmdStr, srcIP, dstIP string
+	// trigger a flow with iperf
+	if !isIPv6 {
+		srcIP = podAIPs.ipv4.String()
+		dstIP = podBIPs.ipv4.String()
+		cmdStr = fmt.Sprintf("iperf3 -c %s -t %d", dstIP, iperfTimeSecShort)
+	} else {
+		srcIP = podAIPs.ipv6.String()
+		dstIP = podBIPs.ipv6.String()
+		cmdStr = fmt.Sprintf("iperf3 -6 -c %s -t %d", dstIP, iperfTimeSecShort)
+	}
+	stdout, _, err := data.runCommandFromPod(testNamespace, "perftest-a", "perftool", []string{"bash", "-c", cmdStr})
+	if err != nil {
+		t.Errorf("Error when running iperf3 client: %v", err)
+	}
+	_, srcPort, dstPort := getBandwidthAndPorts(stdout)
+
+	// run antctl command on flow aggregator to get flow records
+	var command []string
+	args := []string{"get", "flowrecords", "-o", "json", "--srcip", srcIP, "--srcport", srcPort}
+	if testOptions.enableCoverage {
+		antctlCovArgs := antctlCoverageArgs("antctl-coverage")
+		command = append(antctlCovArgs, args...)
+	} else {
+		command = append([]string{"antctl"}, args...)
+	}
+	t.Logf("Run command: %s", command)
+	stdout, stderr, err := runAntctl(podName, command, data)
+	if err != nil {
+		t.Errorf("Error when running 'antctl get flowrecords -o json' from %s: %v\n%s", podName, err, antctlOutput(stdout, stderr))
```

Comment:
could you use `require.NoError` here and above as well, for consistency?

Reply:
updated